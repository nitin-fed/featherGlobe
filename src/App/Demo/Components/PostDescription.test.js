import React from 'react'
import renderer from 'react-test-renderer'
import { PostDescription } from './PostDescription';


describe('Post Description', () => {
    const props = {
        show: true,
        onBackdropClicked: jest.fn(),
        selectedPost: 1
    }
    test('Should render', () => {
        const component = renderer.create(<PostDescription {...props} />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })

    test('Should render if show === false', () => {
        const newProps = {
            ...props,
            show: false
        }
        const component = renderer.create(<PostDescription {...newProps} />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })


});