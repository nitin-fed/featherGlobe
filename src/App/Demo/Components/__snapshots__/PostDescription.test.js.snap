// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Description Should render 1`] = `
<div>
  <div
    className="Backdrop"
    onClick={[MockFunction]}
  />
  <div
    className="postDescription"
    style={
      Object {
        "top": "200px",
      }
    }
  >
    <h1 />
    <h3 />
    <p />
    <button>
      Delete
    </button>
  </div>
</div>
`;

exports[`Post Description Should render if show === false 1`] = `null`;
